src/components/nav-links-sidebar.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
+
+import Link from 'next/link';
+import { usePathname } from 'next/navigation';
+
+import { Button, buttonVariants } from '@/components/ui/button';
+
+import { navLinks } from '@/config/nav-links';
+
+import { cn } from '@/lib/utils';
+
+import { useSidebarToggle } from '@/hooks/use-sidebar-toggle';
+import { useStore } from '@/hooks/use-store';
+
+import { LogOut } from 'lucide-react';
+
+export const NavLinksSidebar = () => {
+  const pathname = usePathname();
+  const sidebar = useStore(useSidebarToggle, (state) => state);
+
+  return (
+    <>
+      <div className='flex w-full flex-col gap-3'>
+        {navLinks.map(({ href, icon: Icon, label }, index) => (
+          <div key={href + index} className='w-full'>
+            <Button
+              className={cn(
+                'mb-1 h-11 w-full justify-start bg-transparent hover:bg-brand-400/15',
+                pathname.match(href) &&
+                  'bg-brand-500/25 text-brand-100 hover:bg-brand-600/50 hover:text-brand-100'
+              )}
+              asChild
+            >
+              <Link href={href}>
+                <span className={cn(!sidebar?.isOpen ? '' : 'mr-4')}>
+                  <Icon size={20} />
+                </span>
+                <p
+                  className={cn(
+                    'max-w-[200px] truncate text-base',
+                    sidebar?.isOpen === false
+                      ? '-translate-x-96 opacity-0'
```

Comment:
Deixar a mesma sugestão do outro comentário...

```suggestion
                    !sidebar?.isOpen
```

Reply:
resolved on https://github.com/Super-Patos-Hackaton/frontend/pull/22/commits/97a537da53af499c41603545094d3fb3c784d76f!